.codespellrc

```diff
@@ -47,3 +47,4 @@ ignore-words-list =
   tolen,
   UE,
   WRON,
+  infor,
```

Comment:
@tkaratapanis I think you missed the alphabetic order, not a big issue, but since it is already organized it is better to keep this way. I did it yesterday when I added LCALL word.